app/api/utility.py

```diff
@@ -183,15 +183,17 @@ def create_query(
                     nb:hasLabel ?dataset_name;
                     nb:hasSamples ?subject.
             ?subject a nb:Subject;
-                    nb:hasLabel ?sub_id;
-                    nb:hasSession ?session;
-                    nb:hasSession/nb:hasAcquisition/nb:hasContrastType ?image_modal.
-            ?session nb:hasLabel ?session_id.
+                    nb:hasLabel ?sub_id.
             OPTIONAL {{
-                ?dataset_uuid nb:hasPortalURI ?dataset_portal_uri.
+                ?subject nb:hasSession ?session;
+                         nb:hasSession/nb:hasAcquisition/nb:hasContrastType ?image_modal.
+                ?session nb:hasLabel ?session_id.
+                OPTIONAL {{
+                    ?session nb:hasFilePath ?session_file_path.
+                }}
```

Comment:
does that need to be a nested OPTIONAL? i.e. doesn't every session have a filePath?

Reply:
> doesn't every session have a filePath?

I think this is the case for all our graphs right now, but I kept this in since I noticed in our data model the file path attribute is technically optional:
https://github.com/neurobagel/bagel-cli/blob/4da00b6db4cce30d40f101c0c4e17be25db3828f/bagel/models.py#L52-L56

Once we support purely phenotypic sessions, I think this `OPTIONAL` will also become relevant. I've added a note on this in the related issue:
- https://github.com/neurobagel/bagel-cli/issues/225